ledger-service/jwt/src/test/scala/com/digitalasset/jwt/JwksSpec.scala

```diff
@@ -0,0 +1,112 @@
+// Copyright (c) 2019 The DAML Authors. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.digitalasset.jwt
+
+import java.net.InetSocketAddress
+
+import com.sun.net.httpserver.{HttpExchange, HttpHandler, HttpServer}
+import java.security.KeyPairGenerator
+import java.security.interfaces.{RSAPrivateKey, RSAPublicKey}
+
+import com.digitalasset.jwt.domain.DecodedJwt
+import org.scalatest.{FlatSpec, Matchers}
+import scalaz.syntax.show._
+
+/** Helper to create a HTTP server that serves a constant response on the "/result" URL */
+object SimpleHttpServer {
```

Comment:
As `SimpleHttpServer` is a fairly generic name I'd suggest making this object private (you probably won't need it for other tests given how specific its implementation is).